src/styles/main.scss

```
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
+@import "../styles/utils/_reset";
```

fix imports everywhere
```suggestion
@import "./utils/_reset";
```

---

done